[PRIME-Features] Prime pack: Antag Thief, Pet Thief, Collection, Pets pack, Pets holder

PhantornRU opened the pull request:
Перенесено:
https://github.com/ss220-space/Paradise/pull/2058

PhantornRU committed:
Новый антаг: Вор. Новые цели на воровство, воровство структур, животных, ношение животных

PhantornRU committed:
Новый антаг: Вор. Новые цели на воровство, воровство структур, животных, ношение животных

PhantornRU committed:
Набор гильдии вора

PhantornRU committed:
Набор гильдии вора

PhantornRU committed:
TGUI с выбором интерфейса, спавн наборов

PhantornRU committed:
TGUI с выбором интерфейса, спавн наборов

PhantornRU committed:
Мультииспользование, ТГУИ описание набора в интерфейсе

PhantornRU committed:
Мультииспользование, ТГУИ описание набора в интерфейсе

PhantornRU committed:
Заказ хомяков в карго

PhantornRU committed:
Заказ хомяков в карго

PhantornRU committed:
Merge remote-tracking branch 'upstream/master220' into antag_thief

PhantornRU committed:
Решение конфликта

PhantornRU committed:
Решение конфликта

PhantornRU committed:
Добавление животных и новых целей на карты

PhantornRU committed:
Добавление животных и новых целей на карты

PhantornRU committed:
Update TGUI

PhantornRU committed:
Цели для кражи

PhantornRU committed:
Цели для кражи

PhantornRU committed:
select target новых целей через трейтор панель

PhantornRU committed:
select target новых целей через трейтор панель

PhantornRU committed:
Пинпоинтер, цель на деньги и коллекционирование

PhantornRU committed:
Пинпоинтер, цель на деньги и коллекционирование

PhantornRU committed:
Корректировка обнаружения цели

PhantornRU committed:
Корректировка обнаружения цели

PhantornRU committed:
Дополнительные питомцы: Гусенок, Фенек, Слизнекот, Крысы, Ящерицы, Моль

PhantornRU committed:
Дополнительные питомцы: Гусенок, Фенек, Слизнекот, Крысы, Ящерицы, Моль

PhantornRU committed:
Питомцы: Рататуй, Реми, собака психолога

PhantornRU committed:
Питомцы: Рататуй, Реми, собака психолога

PhantornRU committed:
Дополнительные звуки часто встречаемых животных

PhantornRU committed:
Дополнительные звуки часто встречаемых животных

PhantornRU committed:
Корректировка звуков

PhantornRU committed:
Корректировка звуков

PhantornRU committed:
Доделаны цели, починен пинпоинтер под цели, мелкие правки холдеров, размещение недостающих фигурок

PhantornRU committed:
Доделаны цели, починен пинпоинтер под цели, мелкие правки холдеров, размещение недостающих фигурок

PhantornRU committed:
Лягушка, рефактор иконок, переноска, монета, пинпоинтер фиксы коллекций

PhantornRU committed:
Лягушка, рефактор иконок, переноска, монета, пинпоинтер фиксы коллекций

PhantornRU committed:
Переноска с доп. цветами, рефактор коллекционирования

PhantornRU committed:
Переноска с доп. цветами, рефактор коллекционирования

PhantornRU commented:
ПР пока не трогать. 
Меня ждет развлечение с конфликтами, а также чутка доделать добавление в режимы самих воров и мелкие фиксы, которые заметил в процессе тестирования.

PhantornRU committed:
Убрал дубликат, вернул спейс кота к котам

PhantornRU committed:
Убрал дубликат, вернул спейс кота к котам

VasilevAY commented:
> ПР пока не трогать. 
> Меня ждет развлечение с конфликтами, а также чутка доделать добавление в режимы самих воров и мелкие фиксы, которые заметил в процессе тестирования.

Лейбл "не мерджить" поставить?
@PhantornRU

PhantornRU committed:
Закидываем антага в режим тритора

PhantornRU committed:
Закидываем антага в режим тритора

PhantornRU commented:
@VasilevAY, он всеравно в драфте, пока бестолку.

VasilevAY commented:
> @VasilevAY, он всеравно в драфте, пока бестолку.
> 

Если что - пингуй

PhantornRU committed:
Правка пути кота

PhantornRU committed:
Правка пути кота

dj-34 commented:
Страшно представить как это не должно конфликтовать с апдейтами мейна (особенно карты), надеюсь залив апдейта на все ветки и отключение антага на мейне предусмотрен? 🤷‍♂️

PhantornRU committed:
Merge remote-tracking branch 'upstream/master220' into antag_thief

PhantornRU commented:
> Страшно представить как это не должно конфликтовать с апдейтами мейна (особенно карты), надеюсь залив апдейта на все ветки и отключение антага на мейне предусмотрен? 🤷‍♂️

а нахуа он на мейне? Проведете голосование - будет на мейне. 
А для прайма можно просто резать карту Дельты, на ней и не играют.

PhantornRU committed:
Заного заполняем Гипериаду из-за конфликта слияния

PhantornRU committed:
Заного заполняем Гипериаду из-за конфликта слияния

Den109G commented:
Типичный Пупс.
\> Новый Антаг, считай дополнение к предателю или генокраду
\> Трогаем вообще **все** файлы игры, включая цели станции

:Chad:

PhantornRU committed:
Забытые структуры и питомцы из-за конфликта, докинул обратно

PhantornRU committed:
Забытые структуры и питомцы из-за конфликта, докинул обратно

PhantornRU commented:
> Типичный Пупс. > Новый Антаг, считай дополнение к предателю или генокраду > Трогаем вообще **все** файлы игры, включая цели станции
> 
> :Chad:

Если б не надо было воровать животных и структуры, вообще бы дальше антага никуда не лез

Den109G commented:
> Если б не надо было воровать животных и структуры, вообще бы дальше антага никуда не лез

А превращение в цельку антага после мага - утверждено с заказчиком?
А выпадение цельки вора с харвестера?

Тыж понимаешь, seems something's fishy here.

PhantornRU committed:
Мелкие правки, ТГУИ конфликт, лягушки добавлены в infestation

PhantornRU committed:
Мелкие правки, ТГУИ конфликт, лягушки добавлены в infestation

PhantornRU commented:
А так животных хоть не так тоскливо воровать. Ибо я до сих пор помню какого это попытаться "своровать" активно сопротивляющегося хряка, за которого играет игрок.
Там еще в планах было дать возможность связывать их и замедлять болой... Но пока отложил. А то я бы точно охуел.

Конфликты решил.
В планах мелкие правки животным, пинпоинтеру в коллекционировании пофиксить отсутствие одежды, добавить отсчет для побега из переноски животному (а то прогрессбар не отрисовывается из-за того что животное в content'ах).
Снизить скорость пейперворку и добавить улитке "склизкий слез" оставляемый под собой.
В общем совсем необязательные мелочи.

Возможно в режимы добавить? Сделать Autothief наподобии Autotraitor для совсем лоупопов? С муней позже обсужу если это нужно будет.

PhantornRU commented:
> > Если б не надо было воровать животных и структуры, вообще бы дальше антага никуда не лез
> 
> А превращение в цельку антага после мага - утверждено с заказчиком? А выпадение цельки вора с харвестера?
> 
> Тыж понимаешь, seems something's fishy here.

В цельку антага никто не превращается. Из харвестера не выпадает целька.
Ну и да, всё обсужено, даже во время стримов И В ТОМ ЧИСЛЕ ЛИЧЕК вопросы задавал с ответами "нужно/не нужно".

Завтра уже буду вносить правки если что-то будет не устраивать в итоге, либо найдутся неточности. А пока пойду спать.

dj-34 commented:
> > Страшно представить как это не должно конфликтовать с апдейтами мейна (особенно карты), надеюсь залив апдейта на все ветки и отключение антага на мейне предусмотрен? 🤷‍♂️
> 
> а нахуа он на мейне? Проведете голосование - будет на мейне. А для прайма можно просто резать карту Дельты, на ней и не играют.

А о стягивании апдейтов ты подумал? Удачи Азизу исправлять конфликты. То что Дельта не часто играется на Прайме не значит что она не играется там вообще, тоже самое с гейтом - если его будут изменять, то пойдет все по пизде

dj-34 commented:
А, ну да. Еще стоит подметить что апдейтов Кибериады это тоже касается, мерж конфликтыыыыыыыыыыыыыыыыыыыыыыыыыыыыыыыыыыыыыыыыыыыыыыыыыыыыыыыыыыыыыыыыыыыыы заебисссссс

Bizzonium committed:
Merge branch 'whitelist220' into antag_thief

dj-34 commented on _maps/map_files/cyberiad/cyberiad.dmm:
```diff
@@ -13672,13 +13683,19 @@
+	pixel_x = 9;
+	pixel_y = 6
+	},
+/obj/item/clothing/head/helmet/skull{
+	desc = "Бедный Йорик...";
+	name = "Йорик"
+	},
```
зачем? почему? нахуя?
![Снимок](https://user-images.githubusercontent.com/20109643/218836658-29381d1e-96fd-4e8f-9448-90b9fd78117f.PNG)

PhantornRU replied:
Изначально ради статуэтки которая находится в целях. А еще да, Йорик будет в списке предметов для пиздинга, заранее его расположил.
Эти Йорики на обеих картах теперь.
Взамен выпилил **дубликат** молотка (он и так есть в суде и им никто не пользовался, только разве что бесоебить)

tema5190 commented:
Внешне выглядит очень интересно и аккуратно, пупс базовый чел

PhantornRU committed:
traitorthief + CONFIG

PhantornRU committed:
traitorthief + CONFIG

PhantornRU committed:
CONFIGS + вампиры-воры, вампиры-триторы-воры

PhantornRU committed:
CONFIGS + вампиры-воры, вампиры-триторы-воры

PhantornRU committed:
Решение конфликтов на карте, правки, новые цели для воровства

PhantornRU committed:
Merge branch 'antag_thief' of https://github.com/PhantornRU/Paradise into antag_thief

PhantornRU committed:
Фиксы, решение мерж конфликтов с картой, мелкие правки, лого

перебалансировка газовика
Убрал дубликаты из-за мерджей
Опять дубликаты, фиксы, лого в инструкции,
улитка теперь оставляет склизкий след,

dj-34 commented:
Вижу ты с конфигами начал что-то делать, если ты делаешь для общей ветки (что скорее всего будет необходимым, изза изменений карт и остальных строчек кода (если лить только на вайтлист, то пизда будет мерж конфликтов, я надеюсь это все понимают)), то тогда нужно будет выпилить изменения с Дельты (чтоб мейновцев не касалось это изменение) и допустим закинуть отдельно драфт ПРом, если кто-то предложку сделает на мейны.

PhantornRU commented:
> Вижу ты с конфигами начал что-то делать

Так это даже не полноценный конфиг. Это конфиг-пример же.
Полноценный конфиг уже отдельно с Азизом делается.
Ну и касаемо изменений. У Прайма и Мейна же и так конфиги отличаются.

PhantornRU committed:
Мелкофикс носителя

PhantornRU committed:
Правки от Муни по режимам

PhantornRU committed:
SWITCH case для режимов +++

PhantornRU commented:
Мы переехали отсюда по причине спагетти-мерджа.
